VS2015/Dockerfile.vs2015

```diff
@@ -0,0 +1,18 @@
+FROM microsoft/windowsservercore:10.0.14393.2007
+
+RUN @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+
+RUN powershell -NoProfile -InputFormat None -Command \
+  choco install git ninja cmake jdk8 vcbuildtools python vim -y
```

Comment:
Why is jdk8 required?

Reply:
This is for the Jenkins agent connection.